changelog/v0.2.3/deprecate-ExtAuthPlugin-interface.yaml

```diff
@@ -0,0 +1,5 @@
+changelog:
+- type: FIX
```

Comment:
I wanted to ensure this change was visible to users, despite it being non-functional. Open to feedback on another type that would make sense here.